### ISSUE TITLE
Does not rename imports with `@Nolint`

### ISSUE THREAD

**Issue by csilvers** [Khan/slicker#1]
If we are renaming email_models to email_classes, then we should know to rename
`import email_models  # @Nolint(unused)` to `import email_classes # @Nolint(unused)`.

But we don't.  The reason is that we rename imports by removing the old one and then adding some new one that provides all the needed symbols.  But we don't remove the old one because of the `@Nolint`, and we don't add the new one because nothing needs it (since it's unused).

I don't know the best way of handling this.

**Comment by benjaminjkraft** [comment]
Yeah, it's a tricky issue.  (Many of the same concerns apply to `import *`, although there there's even more we'll mess up.)  In the case where we're moving a symbol, I think the best slicker can do is to tell the user to handle it.  But you're right that when moving a module we should be able to do better.

I think the way to handle this might be something like:
- when looking for references, also look for unused imports, and add a new one if we had an old one.
- when removing imports, if we see an unused import of exactly the moved module (or, for that matter, symbol, although I don't know why you'd do that), remove it.
It's not the cleanest conceptually, though.

**Comment by csilvers** [comment]
I think asking the user to handle it is fine -- this should be rare.  But we don't even do that right now!  I'm not even sure how we'd detect this case.

**Comment by benjaminjkraft** [comment]
Sorry -- by asking the user I meant logging a warning, which I guess we do do.  Maybe that warning has too many false positives to be useful currently?
